subt/docker/robotika/rosconsole.config

```diff
@@ -4,7 +4,7 @@
 #   You can define your own by e.g. copying this file and setting
 #   ROSCONSOLE_CONFIG_FILE (in your environment) to point to the new file
 #
-log4j.logger.ros=DEBUG
+log4j.logger.ros=FATAL
```

Comment:
The DEBUG setting is intentional. We need to have information about ROS dropping messages.

Reply:
Ok.